qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Asynchronous API
+weight: 18
+---
+
+# Using Qdrant asynchronously
+
+Databases are often launched as separate services and are accessed via a network. All the interactions with them are IO-bound and can 
+be performed asynchronously so as not to waste time actively waiting for a server response. If you use Python, that is achieved by 
+using `async/await` syntax. That lets the interpreter switch to another task while waiting for a response from the server.
```

Comment:
Make a link

Reply:
This was a me learning review in Github error. It should have been for 'async/await'

Reply:
Should we link the Python docs? For example, https://docs.python.org/3/library/asyncio-task.html? I believe if someone gets that far, to a tutorial about async API, they should be already familiar with that syntax